winconfig.h

```diff
@@ -20,10 +20,9 @@
 /* @ENABLE_DEFINES@ */
 /* End configure ENABLE_DEFINES picked up from cygconfig.h or other external source, if available */
 
-#if defined(ENABLE_HYBRID_SUSPEND)
 /* Windows MSVC builds defaults to preemptive suspend. Disable ENABLE_HYBRID_SUSPEND defines. */
+/* #undef never needs an ifdef guard, it works unconditionally without warning. */
```

Comment:
this comment belongs to the commit message. doesn't add any value in the source code.

Reply:
I think it helps people learn the language and the rationale of the following line, when they see the undef w/ ifndef.

Some projects comment everything "not obvious", where "not obvious" seems to be "almost any change, because it was wrong-ish the first time, so it must be not obvious".

There is a balance to be found, *somewhere*.
"not obvious" is also subject to "whatever the reader already knows", which is almost different.